tools/create-release-candidate.sh

```diff
@@ -113,4 +114,4 @@ echo
 echo
 echo "Consider running the test babysitter using the pull request number from above:"
 echo
-echo "babysit.py --pr ${PR_NUM} --all -c 1"
+echo "tools/cloud-build/babysit/run --pr <PR_NUM> --all -c 1 --project <team-project>"
```

Comment:
Please remove ` --project <team-project>` , AFAIK it doesn't add to usability.

Reply:
Done